scripts/circleci-run-tests.sh

```diff
@@ -18,6 +18,10 @@ then
     # network tets run inside a container with docker socket mapped in the container.
     docker-compose --file=.circleci/images/docker-compose.yml run integrationTests
 else
+    # ask circleci to split tests by timing.
+    echo "Asking circleci to pick tests based on timings..."
+    export TEST_FILES=$(circleci tests glob "$PWD/test/integration/spec/**/*.js" | circleci tests split --split-by=timings)
```

Comment:
@makarandp0 Does this make sure that test files are split evenly based on execution time?

Reply:
Yes, that what the documentation seem to imply 

> On each successful run of a test suite, CircleCI saves timings data to a directory specified by the path in the store_test_results step. If you do not use store_test_results, there will be no timing data available for splitting your tests.
> To split by test timings, use the --split-by flag with the timings split type.

So far I have not seen it doing good Job. I was hoping to see connect.js not clubbed with other files, but I suspect that's because it does not have sufficient timing data to go by. But even if the split was not timing based, it seems to do a reasonable job.